src/main/java/Duke.java

```diff
@@ -1,10 +1,103 @@
+
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner UserInput = new Scanner(System.in);
+        int listCounter = 0;
+        Task[] myTasks = new Task[100];
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Duke\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n");
+
+        String command = UserInput.nextLine();  // scan user input
+
+        while (!command.equals("bye")) {     // prog doesnt end unless "bye"
+            if (command.equals("list")){        // show list
+                System.out.println("____________________________________________________________\n" +
+                        "Here are the tasks in your list:");
+                for (int i = 0; i<listCounter; i++) {
+                    System.out.println( (i+1) + "." + myTasks[i]);
+                }
+
+            }
+            else if (command.startsWith("done")){       // mark tick
```

Comment:
If I remember correctly, the `else if` part should be placed on the same line with the closing bracket of the if block above it.